database/entity.go

```diff
@@ -0,0 +1,87 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+type Entity struct {
+	Id   int64
+	Name string
+}
+
+func (d *Database) CreateEntity(entity *Entity) error {
+	query := "INSERT INTO entities (name) VALUES (?) RETURNING id"
+	row := d.QueryRow(query, entity.Name)
```

Comment:
Does `QueryRow` execute `INSERT` statements?

Reply:
It works, so yes